src/test/java/com/bitmovin/api/examples/CreateEncodingWithTimecode.java

```diff
@@ -0,0 +1,210 @@
+import com.bitmovin.api.encoding.enums.CloudRegion;
+import com.bitmovin.api.encoding.enums.StreamSelectionMode;
+import com.bitmovin.api.encoding.inputs.HttpsInput;
+import com.bitmovin.api.encoding.outputs.GcsOutput;
+import com.bitmovin.api.encoding.outputs.Output;
+import com.bitmovin.api.encoding.outputs.S3Output;
+import com.bitmovin.api.encoding.status.Task;
+import com.bitmovin.api.enums.Status;
+import com.bitmovin.api.exceptions.BitmovinApiException;
+import com.bitmovin.api.http.RestException;
+import com.bitmovin.api.webhooks.Webhook;
+import com.bitmovin.api.webhooks.enums.WebhookHttpMethod;
+import com.bitmovin.api.webhooks.enums.WebhookType;
+import com.mashape.unirest.http.exceptions.UnirestException;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Created by Roland Kersche on 30.05.17.
+ */
+public class CreateEncodingWithTimecode
+{
+    private static String API_KEY = "<INSERT_YOUR_APIKEY>";
+
+    private static CloudRegion CLOUD_REGION = CloudRegion.AWS_EU_WEST_1;
+    private static String HTTPS_INPUT_HOST = "<INSERT_YOUR_HTTP_HOST>"; // ex.: storage.googleapis.com/
+    private static String HTTPS_INPUT_PATH = "<INSERT_YOUR_PATH_TO_INPUT_FILE>";
+    private static String S3_OUTPUT_ACCESSKEY = "<INSERT_YOUR_ACCESSKEY>";
+    private static String S3_OUTPUT_SECRET_KEY = "<INSERT_YOUR_SECRETKEY>";
+    private static String S3_OUTPUT_BUCKET_NAME = "BUCKET_NAME";
+    private static String OUTPUT_BASE_PATH = "path/to/your/outputs/" + new Date().getTime();
+
+    private static BitmovinApi bitmovinApi;
+
+    @Test
+    public void testEncoding() throws IOException, BitmovinApiException, UnirestException, URISyntaxException, RestException, InterruptedException
+    {
+        bitmovinApi = new BitmovinApi(API_KEY);
+        Encoding encoding = new Encoding();
+        encoding.setName("Encoding Java Example - Add TimeCode to MP4Muxing");
+        encoding.setCloudRegion(CLOUD_REGION);
+        encoding = bitmovinApi.encoding.create(encoding);
+
+        HttpsInput input = new HttpsInput();
+        input.setHost(HTTPS_INPUT_HOST);
+        input = bitmovinApi.input.https.create(input);
+
+        S3Output output = new S3Output();
+        output.setAccessKey(S3_OUTPUT_ACCESSKEY);
+        output.setSecretKey(S3_OUTPUT_SECRET_KEY);
+        output.setBucketName(S3_OUTPUT_BUCKET_NAME);
+        output = bitmovinApi.output.s3.create(output);
+
+        AACAudioConfig aacConfiguration = new AACAudioConfig();
+        aacConfiguration.setBitrate(96000L);
+        aacConfiguration.setRate(48000f);
+        aacConfiguration = bitmovinApi.configuration.audioAAC.create(aacConfiguration);
+
+        H264VideoConfiguration videoConfiguration240p = new H264VideoConfiguration();
+        videoConfiguration240p.setHeight(240);
```

Comment:
Please also change that so that we only have a configuration array at the beginning and the configurations, streams, muxings, ... generated dynamically. As e.g. here: https://github.com/bitmovin/bitmovin-python/blob/master/examples/encoding/create_simple_encoding_crop_filter.py#L25